docs/using-the-snowplow-console/managing-users/index.md

```diff
@@ -2,6 +2,9 @@
 title: "Managing users"
 date: "2020-02-15"
 sidebar_position: 30
+sidebar_custom_props:
+  offerings:
+    - enterprise
 ---
 
 To manage users in console, click 'Manage users' in the navigation bar.
```

Comment:
Maybe add a docs card to the bottom of this page, as it's not clear there are children (and we get that feedback a lot...)

Reply:
Actually I am going to empty this page as it adds nothing (aside from some outdated screenshots) to what’s inside it.